pkg/parse/parser.go

```diff
@@ -92,9 +93,7 @@ func UnmarshalAny(buf []byte, details *AnyDetails) (*ubp.Blueprint, error) {
 
 	// Try BP JSON
 	bpData := new(bp.Blueprint)
-	dec := json.NewDecoder(bytes.NewReader(buf))
-	dec.DisallowUnknownFields()
-	err = dec.Decode(bpData)
+	err = json.Unmarshal(buf, bpData, json.DefaultOptionsV2())
```

Comment:
This is the place where the new (experimental) V2 JSON parser is used instead of the Go built-in parser.